spec/AsyncCheckerSpec.php

```diff
@@ -62,4 +62,20 @@
             expect($results[0]->getStatusCode())->toEqual($expected[0]->getStatusCode());
         });
     });
+
```

Comment:
I'd like a test adding for the command line options to ensure they're configured correcly I think

Reply:
Added the response timeout, not sure yet how to set up the connect timeout test. Might set up a small @reactphp project up that listens on a specific port to do something like that

Reply:
I'm okay with the test being left for now, and an issue being filed to add it at a later time.

Reply:
Created a new issue for that as https://github.com/Brunty/cigar/issues/34